docs/docusaurus.config.ts

```diff
@@ -77,6 +77,19 @@ const config: Config = {
   ],
   themes: ["@easyops-cn/docusaurus-search-local"],
 
+  plugins: [
+    [
+      "@docusaurus/plugin-client-redirects",
+      {
+        redirects: [
+          { to: "/docs/credentials/add-credential", from: ["/docs/environments/credentials/add-credential"] },
+          { to: "/docs/credentials/list-credentials", from: ["/docs/environments/credentials/list-credentials"] },
+          { to: "/docs/credentials/view-credential", from: ["/docs/environments/credentials/view-credential"] }
```

Comment:
@AhmadYasser1 perhaps, for better DX and readability, define `from` before `to`? I believe that could bring a more natural flow while reading the code. What do you think?